src/views/admin/PersonDetailsView.vue

```diff
@@ -443,10 +447,24 @@
     }),
   );
 
-  // Validation schema for the form for changing KopersNr
+  // Validation schema for the form for changing person metadata
   const changePersonInfoValidationSchema: TypedSchema = toTypedSchema(
     object({
-      selectedKopersNrPersonInfo: string().required(t('admin.person.rules.kopersNr.required')),
+      selectedKopersNrPersonInfo: string().when([], {
```

Comment:
```suggestion
      selectedKopersNr: string().when([], {
```

Reply:
selectedKopersNr haben wir eigentlich schon. Das ist die Variable für Kopers bei Schulzuordnung hinzufügen.

Ich habe jetzt das in selectedKopersNrMetaData umbennant, passt das so?